lib/pbench/server/auth/__init__.py

```diff
@@ -71,20 +70,38 @@ def del_param_headers(self, key: str):
         """
         del self.headers[key]
 
-    def get_well_known(self) -> JSON:
+    def set_well_known_endpoints(self) -> JSON:
```

Comment:
I think this should be done implicitly as part of opening a connection, rather than relying on an outside caller to know about it...